src/python/examples/streamlined_example.py

```diff
@@ -0,0 +1,24 @@
+import tritonserver
```

Comment:
This file is probably a good snippet for a `README.md` about this feature. 

Can we add docs?